conf/incliva.config

```diff
@@ -0,0 +1,52 @@
+// Profile details
+params {
+    config_profile_description = 'INCLIVA Health Research Institute profile for nf-core.'
+    config_profile_contact = 'Sheila Zúñiga Trejos - bioinfo@incliva.es'
```

Comment:
Shouldn't this be you @nevinwu ?

Reply:
Actually, that is the boss/coordinator of where I'm working at the moment (INCLIVA's bioinformatics unit). Unfortunately, I will no longer be working here starting on September. So, I thought it would be better to set her info as "profile_contact" since she will be staying here for a long time.

Reply:
Fair enough!